src/solidity-frontend/solidity_grammar.cpp

```diff
@@ -460,27 +553,9 @@ ExpressionT get_expr_operator_t(const nlohmann::json &expr, bool uo_pre)
   {
     return BO_Rem;
   }
-  else if(expr["operator"] == "--")
-  {
-    if(uo_pre)
-    {
-      return UO_PreDec;
-    }
-    else
-    {
-      assert(!"Unsupported - UO_PostDec");
-    }
-  }
-  else if(expr["operator"] == "++")
+  else if(expr["operator"] == "&&")
```

Comment:
That seems to be the same case as just below.

Reply:
This is due to a bad merge, will remove.